<launch>
	<!-- Alter back_lbz PID
	<param name="back_lbz_position_controller/pid/d" type="double" value="300" />
	<param name="back_lbz_position_controller/pid/p" type="double" value="100" />
	<node name="reload_back_lbz" pkg="C45_PostureControl" type="reload_back_lbz_controller.sh" />	-->
	<!--
	<param name="PID_gains/P" type="double" value="0.00" />
	<param name="PID_gains/I" type="double" value="-0.00" />
	<param name="PID_gains/D" type="double" value="-0.02" />
	-->
	<node name="PoseController" pkg="PoseController" type="PoseController" cwd="node" output="screen" respawn="true"/>
	<node name="move_back_lbz_neck_ay" pkg="PoseController" type="move_back_lbz_neck_ay" cwd="node" output="screen" respawn="true"/>
	
	<!--<node name="C45_PostureControl_com_error" pkg="C45_PostureControl" type="C45_PostureControl_com_error" cwd="node" output="screen" />-->
	<!--<node name="C45_PostureControl_back_lbz_poller_service" pkg="C45_PostureControl" type="C45_PostureControl_back_lbz_poller_service" cwd="node" output="screen" />-->
	<node name="C45_PostureControl" pkg="C45_PostureControl" type="C45_PostureControl" cwd="node" output="screen" respawn="true"/>
	<node name="C45_TaskStart" pkg="C45_PostureControl" type="C45_TaskStart" cwd="node" output="screen" />
	<node name="C45_TaskStop" pkg="C45_PostureControl" type="C45_TaskStop" cwd="node" output="screen" />
	<node name="C45_Behavior" pkg="C45_PostureControl" type="C45_Behavior" cwd="node" output="screen" />
	<!--<node name="C45_PostureControl_maintain_stability" pkg="C45_PostureControl" type="C45_PostureControl_maintain_stability" cwd="node" output="screen" />-->
	<!--<node name="C45_PostureControl" pkg="C45_PostureControl" type="C45_PostureControl" cwd="node"/>-->
	
</launch>
